proposals/v3/server.md

```diff
@@ -0,0 +1,13 @@
+# Synse Server
+
+Below are a collection of notes that generally apply to Synse Server
+that will need to be formalized more:
+
+- the vast majority of pre-processing of data should not be done in
+  synse server.
+  - this largely applies to the alias endpoints; not all LED devices may
```

Comment:
We are not keeping the alias endpoints right?

Reply:
right; they'd go away in v3. there is some pre-processing done in other endpoints, but the majority of it happens in the alias endpoints